aten/src/ATen/native/quantized/cpu/qlinear.h

```diff
@@ -18,7 +18,7 @@ class QLinearOnednn final {
       int64_t output_zero_point,
       std::optional<c10::ScalarType> output_dtype,
       std::string_view post_op_name,
-      torch::List<std::optional<at::Scalar>> post_op_args,
+      c10::List<std::optional<at::Scalar>> post_op_args,
```

Comment:
This change is needed otherwise there is an error when compiling torch/csrc/inductor/aoti_torch/shim_common.cpp